_overviews/scala3-macros/tutorial/reflection.md

```diff
@@ -230,6 +230,6 @@ We can make this printer the default if needed
 *Coming soon*
 
 [tasty inspection]: {{ site.scala3ref }}/metaprogramming/tasty-inspect.html
-[reflection doc]: https://dotty.epfl.ch/api/scala/quoted/Quotes$reflectModule.html?query=trait%20reflectModule
+[reflection doc]: https://scala-lang.org/api/3.x/api/scala/quoted/Quotes$reflectModule.html
```

Comment:
I’m not sure what the `query` parameter was supposed to do. It seems to do nothing on dotty.epfl.ch

Reply:
Scaladoc does not use query paramters at this moment.